lib/dao.js

```diff
@@ -2669,6 +2669,19 @@ DataAccessObject.updateAll = function(where, data, options, cb) {
   this.applyScope(query);
   this.applyProperties(data);
 
+  var doValidate = undefined;
+  if (options.validate === undefined) {
+    if (Model.settings.validateUpdate === undefined) {
```

Comment:
Would you mind using `validateUpdateAll` as the option name? I am personally never sure whether `update` is for a single or multiple records.